ui/src/data-services/hooks/projects/utils.ts

```diff
@@ -13,5 +13,8 @@ export const convertToServerFormData = (fieldValues: any) => {
     data.append('image', '')
   }
 
+  // TODO: Remove console log and include default filters as form data
```

Comment:
The form is working, we just need to pass the values here when the backend is ready.